docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -135,23 +117,23 @@ For more information see reference at the [Hardware specification](https://githu
 |STM32H743|STM32H743|
 |STM32H750|STM32H750|
 
-- find and install the firmware (4.0 or newer) for the Unified Target identified above. The firmware is available from the board type drop-down in configurator. Be aware that after flashing this firmware, the LEDs on your board will not be working - this is normal.
+- find and install the firmware (4.0 or newer) for the Target identified above. The firmware is available from the board type drop-down in configurator. Be aware that after flashing this firmware, the LEDs on your board will not be working - this is normal.
```

Comment:
NONE of the above list of firmware appears in the dropdown menu in the configurator.

![Uploading image.png…]()

Reply:
But it is on the release page OR you can build your own. Will update the comment.

Reply:
I raised this as an issue, see: https://github.com/betaflight/betaflight-configurator/issues/3164